activiti-cloud-api/activiti-cloud-api-process-model/src/main/java/org/activiti/cloud/api/process/model/events/ExtendedCloudProcessRuntimeEvent.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2017-2020 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.activiti.cloud.api.process.model.events;
+
+import org.activiti.api.process.model.ProcessInstance;
+import org.activiti.cloud.api.model.shared.events.CloudRuntimeEvent;
+
+public interface ExtendedCloudProcessRuntimeEvent extends CloudRuntimeEvent<ProcessInstance, ExtendedCloudProcessRuntimeEvent.ExtendedCloudProcessRuntimeEvents> {
```

Comment:
I created a new class for avoiding coupling the core of Activiti with an action that is not part of the engine.
In fact, reusing CloudProcessRuntimeEvent meant introducing `PROCESS_DELETED` event in `org.activiti.api.process.model.events.ProcessRuntimeEvent.ProcessEvents`, but the new event and the user request for deleting a process do not have impact on RB since it deletes every process information once they are completed or cancelled.
The new API has been introduced on RB just because the Query Service could not generate events by architectural design.

Reply:
Done, please see also PR [Activiti#4143](https://github.com/Activiti/Activiti/pull/4143)